test/unit/cog_api/fake/relays_test.exs

```diff
@@ -134,6 +134,20 @@ defmodule CogApi.Fake.RelaysTest do
       assert :ok == Client.relay_delete(relay.id, valid_endpoint)
     end
 
+    it "deletes the association in relay_groups" do
+      relay = Client.relay_create(%{name: "delete me", token: "1234"}, valid_endpoint)
+      |> get_value
+      group = Client.relay_group_create(%{name: "relay_group"}, valid_endpoint) |> get_value
+      {:ok, _} = Client.relay_group_add_relays_by_id(group.id, relay.id, valid_endpoint)
+
+      Client.relay_delete(relay.id, valid_endpoint)
+
+
+      group = Client.relay_group_show(group.id, valid_endpoint) |> get_value
+
+      assert group.relays == []
```

Comment:
Should we add an assertion about this that the `group.relays` does have that relay?